web/gridt/schemas.py

```diff
@@ -24,3 +24,11 @@ class MovementSchema(Schema):
 class ChangePasswordSchema(Schema):
     old_password = fields.Str(required=True)
     new_password = fields.Str(required=True)
+
+
+class RequestPasswordResetSchema(Schema):
+    email = fields.Email(required=True)
+
+class ResetPasswordSchema(Schema):
+    token = fields.Str(required=True)
+    password = fields.Str(required=True)
```

Comment:
Maybe move the JWT logic here.

Reply:
As of yet I have no idea how to implement this with marshmallow.